sqladmin/helpers.py

```diff
@@ -26,6 +27,13 @@
 T = TypeVar("T")
 
 
+class AlertTypeEnum(enum.Enum):
+    success = "success"
+    info = "info"
+    warning = "warning"
+    danger = "danger"
```

Comment:
Why limit the message type? I think this could be just a string.

Reply:
These map to tabler alert types https://tabler.io/docs/components/alerts

Reply:
That's a good point but I believe that this shouldn't be limited by an enumeration.

Even though flash messages have some basic categories, they could have custom categories.

WDYT @aminalaee?

Reply:
Looks like django uses an enum https://docs.djangoproject.com/en/5.0/ref/contrib/messages/#adding-a-message

Reply:
Oh I see...